test/gtest/ucm/malloc_hook.cc

```diff
@@ -56,6 +61,7 @@ class malloc_hook : public ucs::test {
 
 int malloc_hook::small_alloc_count = 1000 / ucs::test_time_multiplier();
 
+#if HAVE_MALLOC_STATES
```

Comment:
can we just skip the test part which calls malloc_get_state/malloc_set_state? wouldn't want to disable malloc hooks tests completely

Reply:
done